podcasts/FeatureFlag.swift

```diff
@@ -77,6 +84,18 @@ enum FeatureFlag: String, CaseIterable {
             return true
         case .newShowNotesEndpoint:
             return true
+        case .episodeFeedArtwork:
+            return true
```

Comment:
I wonder if it would be easier to just define a feature as enabled on TestFlight using a check on the feature flag value itself, rather than defining a new private property?

```suggestion
            return FeatureFlag.isTestFlight ? true : false
```

Reply:
Done in afcef1b18f072a6b5a9bd275f8138ced210e8cad